web/src/test/java/se/inera/intyg/intygstjanst/web/integration/SendCertificateToRecipientResponderImplTest.java

```diff
@@ -195,6 +153,61 @@ public void testSendTestCertificateToRecipientTestCertificateException() throws
         assertEquals("Certificate 'Intygs-id-1234567890' couldn't be sent to recipient because it is a test certificate", response.getResult().getResultText());
     }
 
+    void setup() {
+        request = createRequest();
+        request.getMottagare().setCode("FKASSA");
+    }
+
+    @SneakyThrows
```

Comment:
Is this annotation okay to use for tests or should I handle it in some other way?

Reply:
I haven't used it myself, but for test classes it could be quite useful. Otherwise you have to add "throws"-declarations on the test method, right?

I'm OK if you keep as is.

Reply:
Yes otherwise I have to add all the different exceptions. I'll keep this then!